exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java

```diff
@@ -1,169 +1,179 @@
+	
+	private int getAmountOfPlayers() {
+    	return playersInGame.size();
+	}
+
+	public void playGame(int roll) {
+    	Player player = playersInGame.get(playerLocation);
+		System.out.println(playersInGame.get(playerLocation) + " is the current player");
+		System.out.println("They have rolled a " + roll);
+		
+		if (player.isPlayerInPenaltyBox()) {
+			penaltyCheck(roll);
+		} else {
+			updatePlayerLocation(roll);
+		}
+	}
+
+	private void penaltyCheck(final int result) {
+		Player player = playersInGame.get(playerLocation);
+
+		if (result % PENALTY_CHECK_VALUE != 0) {
+			System.out.println(playersInGame.get(playerLocation) + " is getting out of the penalty box");
+			player.takePlayerOutOfPenaltyBox();
+			isOutOfPenaltyBox = true;
+
+			updatePlayerLocation(result);
+		} else {
+			System.out.println(playersInGame.get(playerLocation) + " is not getting out of the penalty box");
+			isOutOfPenaltyBox = false;
+			}
+	}
+
+	private void updatePlayerLocation(final int roll) {
+		playerOrder[playerLocation] += roll;
+		if (playerOrder[playerLocation] > LAST_POSITION) {
+			playerOrder[playerLocation] -= RESET_POSITION;
+		}
+
+		System.out.println(playersInGame.get(playerLocation)
+				+ "'s new location is "
+				+ playerOrder[playerLocation]);
+		System.out.println("The category is " + getCategory());
+		askQuestion();
+	}
+
+	private void askQuestion() {
+		if (getCategory().equals(CATEGORY_POP))
+			System.out.println(popQuestionsList.removeFirst());
+		if (getCategory().equals(CATEGORY_SCIENCE))
+			System.out.println(scienceQuestionsList.removeFirst());
+		if (getCategory().equals(CATEGORY_SPORTS))
+			System.out.println(sportsQuestionsList.removeFirst());
+		if (getCategory().equals(CATEGORY_ROCK))
+			System.out.println(rockQuestionsList.removeFirst());
+	}
+	
+	
+	private String getCategory() {
+    	String[] boardSpaces = {
+    			CATEGORY_POP, CATEGORY_SCIENCE, CATEGORY_SPORTS, CATEGORY_ROCK,
+				CATEGORY_POP, CATEGORY_SCIENCE, CATEGORY_SPORTS, CATEGORY_ROCK,
+				CATEGORY_POP, CATEGORY_SCIENCE, CATEGORY_SPORTS, CATEGORY_ROCK};
+    	return boardSpaces[playerOrder[playerLocation]];
+	}
+
+	public boolean wasAnsweredCorrectly() {
+		if (playersInPenaltyBox[playerLocation]){
+			if (isOutOfPenaltyBox) {
+				System.out.println("Answer was correct!!!!");
+				return calculateScoreAndDisplay();
+			} else {
+				playerLocation++;
+				resetPlayerLocationToZero();
+				return true;
+			}
+		} else {
+			// Don't touch this spelling mistake upon pain of death
+			// It seems to hold the whole universe together
+			System.out.println("Answer was corrent!!!!");
+			return calculateScoreAndDisplay();
+		}
+	}
+
+	public boolean wasAnsweredIncorrectly(){
+		Player player = playersInGame.get(playerLocation);
+		System.out.println("Question was incorrectly answered");
+		System.out.println(playersInGame.get(playerLocation)+ " was sent to the penalty box");
+
+		player.putPlayerInPenaltyBox();
+		//playersInPenaltyBox[playerLocation] = true;
```

Comment:
I'm assuming this comment was left in by accident and not essential to the code?